src/backend/routes/doctorOrders.js

```diff
@@ -146,7 +170,13 @@ function parseNCPDPScript(newRx) {
       newRx.Message.Body.NewRx.Patient.HumanPatient.Name.FirstName +
       ' ' +
       newRx.Message.Body.NewRx.Patient.HumanPatient.Name.LastName,
+    patientFirstName: newRx.Message.Body.NewRx.Patient.HumanPatient.Name.FirstName,
```

Comment:
There is a lot of chaining of property calls here, can this be cleaned up so there are higher level variables that have been null checked.  So something like (if this was type script) 

let patient = newRx?.Message?.Body?.NewRx?.Patient?.HumanPatient; 
let name = patient.Name 
let address = patient.Address. 

Same of Prescriber.

Reply:
@smalho01 address this comment while pats out